### PR TITLE
Update styling aggrid

### PR DESCRIPTION
**Pull request by dylanmau**
https://timechimp.atlassian.net/jira/software/c/projects/CSR/boards/19/backlog?view=detail&selectedIssue=CSR-3760&issueLimit=100&assignee=63ff4d19314f50881381751a

### CODE REVIEW

File: src/components/data-grid/styles.ts [hunk 1]
```diff
@@ -11,18 +11,18 @@ export const getGridThemeOverrides = (theme: CustomThemeType) => {
       --ag-alpine-active-color: ${theme.colors.primary};
       --ag-foreground-color: ${theme.colors.primaryA};
       --ag-secondary-foreground-color: ${theme.customColors.dark4};
-      --ag-row-hover-color: ${theme.colors.primary100};
-      --ag-selected-row-background-color: ${theme.colors.primary200};
+      --ag-row-hover-color: ${theme.customColors.light7};
```
Comment (hiddeottens) [inline src/components/data-grid/styles.ts:14]: What about the theme coloring?

Comment (hiddeottens) [reply src/components/data-grid/styles.ts:14]:
```suggestion
      --ag-row-hover-color: ${theme.colors.primary50};
```

File: src/components/data-grid/styles.ts [hunk 2]
```diff
@@ -11,18 +11,18 @@ export const getGridThemeOverrides = (theme: CustomThemeType) => {
...
+      --ag-selected-row-background-color: ${theme.colors.primary}0f;
```
Comment (hiddeottens) [inline src/components/data-grid/styles.ts:15]:
```suggestion
      --ag-selected-row-background-color: ${theme.colors.primary100};
```

**Review by hiddeottens — CHANGES_REQUESTED** [review]
some minor styling changes

File: src/components/data-grid/styles.ts (hunk 2 above)
Comment (dylanmau) [reply src/components/data-grid/styles.ts]: Changed it

**Review by hiddeottens — APPROVED** [review]

**Pull request merged**